### PR TITLE
remove unnecessary comments in execinput file

### PR DESCRIPTION
**Pull request by weiyuan-jiang**

### CODE REVIEW

File: src/Applications/LDAS_App/GEOSldas_LDAS.rc [hunk 1]
```diff
@@ -53,13 +53,6 @@ LSM_CHOICE: 1
 # WHITE_FILE: '' 
 
 
-# ---- Surface meteorological forcing: Time step
```
Comment (gmao-rreichle) [inline src/Applications/LDAS_App/GEOSldas_LDAS.rc:56]: I don't think we can just remove this.  While MERRA-2 and GEOS FP forcing use the default (3600s, or 1 hour), other legacy forcing data have different values (e.g., 3 hours). 
Is there a reason why this was removed?

Comment (weiyuan-jiang) [reply src/Applications/LDAS_App/GEOSldas_LDAS.rc:56]: This is in the "required" field

Comment (weiyuan-jiang) [reply src/Applications/LDAS_App/GEOSldas_LDAS.rc:56]: Maybe we can move up the comments

Comment (gmao-rreichle) [reply src/Applications/LDAS_App/GEOSldas_LDAS.rc:56]: Sorry, I didn't catch that this is already put into the *.exec through "ldas_setup" and thus ended up there twice.  Nice catch.  I will edit the comment in "ldas_setup".  Stay tuned.

**Pull request merged**